tests/contrib/validation/cerberus/test_validator_cerberus.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+from unittest import TestCase
+import six
+from slugify import slugify
+
+from spidermon.contrib.validation import CerberusValidator
+
+
+class SchemaTestCaseMetaclass(type):
```

Comment:
It is fine for now, but it would be nice to refactor it to use `pytest`

Reply:
Looking forward to that, adding a card for it.